src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FileAnnotationTabViewModel.java

```diff
@@ -84,7 +89,10 @@ public void notifyNewSelectedFile(Path newFile) {
                 .map(FileAnnotationViewModel::new)
                 .collect(Collectors.toList());
         annotations.setAll(newAnnotations);
-
+        if (annotations.isEmpty())
```

Comment:
Please always use curly braces to the full extend (for if and else here). It is too easy to accidentally introduce a bug when you leave them out.